script.js

```diff
@@ -1,25 +1,36 @@
 var player = 1;
 var jogadas = 0;
 var ha_vencedor = false;
-var valida_posicao =  new Array(0,0,0,0,0,0,0,0,0);
-var marca_posicao = new Array();
+var valida_posicao = [
+                      [false,false,false], // valida_posicao[0][0],valida_posicao[0][1],valida_posicao[0][2],
+                      [false,false,false], // valida_posicao[1][0],valida_posicao[1][1],valida_posicao[1][2],
+                      [false,false,false], // valida_posicao[2][0],valida_posicao[2][1],valida_posicao[2][2],
+];
 
-function clic(posicao_jogada){
+var marca_posicao =  [
+    [false,false,false],
+    [false,false,false], 
+    [false,false,false], 
+];
+
+function clic(coluna,linha){
```

Comment:
The param names are wrong. If you click on the cell labeled as `2, 0`, you'll see that. The cell labeled as `2, 0` stands for "row 2, column 0", alright? But then, when your `click` function is called, you have `coluna = 2` and `linha = 0`. It should be `linha = 2` and `coluna = 0`.

Also, keep in mind what I told about ordering the rows from top to bottom.